backends/bmv2/bmv2.cpp

```diff
@@ -104,12 +104,11 @@ int main(int argc, char *const argv[]) {
     // Generate a PI control plane API for this program if requested.
     if (!options.p4RuntimeFile.isNullOrEmpty()) {
         std::ostream* out = openFile(options.p4RuntimeFile, false);
-        std::ostream *outEntries = nullptr;
+        std::ostream* outEntries = nullptr;
```

Comment:
You and @ChrisDodd are moving the stars around in different directions.
We should settle for one, or make this a don't care, otherwise we'll never converge.

Reply:
It's for the greater good. We are all too aware of what will happen when the stars align.